GroceryStore/InventoryViewController.m

```diff
@@ -0,0 +1,94 @@
+//
+//  InventoryViewController.m
+//  GroceryStore
+//
+//  Created by Jilian Scott on 5/13/15.
+//  Copyright (c) 2015 Vokal. All rights reserved.
+//
+
+#import "InventoryViewController.h"
+#import "Product.h"
+#import "ProductTableViewCell.h"
+#import "ProductTableView.h"
+
+#import "ProductDetailViewController.h"
+
+@interface InventoryViewController () <UITableViewDataSource, UITableViewDelegate>
+
+@property (strong, nonatomic) NSArray *productArray;
+@property (weak, nonatomic) IBOutlet ProductTableView *productTableView;
+
+
+@end
+
+@implementation InventoryViewController
+
+- (void)viewWillAppear:(BOOL)animated {
```

Comment:
per the coding standards, the opening brace `{` should be on the next line. (Objective Clean would catch this also)